src/DssSpell.sol

```diff
@@ -110,13 +110,32 @@ contract DssSpell {
     }
 
     modifier limited {
+        require(nextCastTime() == now, "Outside office hours");
+        _;
+    }
+
+    function nextCastTime() public returns (uint256) {
```

Comment:
can this be a `view`?

Reply:
Yep my bad I had to remove `view` when I was using Debug events. I'll add back.